framework/include/multiapps/TransientMultiApp.h

```diff
@@ -36,53 +36,23 @@ class TransientMultiApp :
 
   virtual ~TransientMultiApp();
 
-  /**
```

Comment:
@permcody removed comments from newly-overrided functions that were copied/pasted from base class functions.  The idea is to reduce the amount of documentation upkeep overhead, i.e. if you change the base class function and update it's documentation, you don't want to have to remember to go update all the child class documentation.